sql/test/01-hardware-profiles.sql

```diff
@@ -1,7 +1,7 @@
 INSERT INTO hardware_product_profile ( product_id, purpose, bios_firmware, cpu_num, cpu_type, 
             dimms_num, ram_total, nics_num, psu_total, rack_unit, usb_num )
        VALUES ( ( SELECT id FROM hardware_product WHERE name = 'S4048-ON' ),
-            'TOR switch', '9.10(0.1P18)', 1, 'Intel Rangeley',
+            'TOR switch', '9.11(2.5)', 1, 'Intel Rangeley',
```

Comment:
Once a migration has been run in a database, it won't be run again. Migrations shouldn't be changed after they have been released.

You did the right thing by making a new migration and doing the update there. 👍 Just lettin' you know how it works.